css/properties/mask.json

```diff
@@ -13,18 +13,27 @@
               },
               {
                 "prefix": "-webkit-",
-                "version_added": true,
+                "version_added": "1",
+                "notes": "The prefixed property can be used with SVG and HTML with a slightly different syntax, which allows setting the non-standard <a href='https://developer.mozilla.org/docs/Web/CSS/-webkit-mask-attachment'><code>-webkit-mask-attachment</code></a> property."
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "18",
+                "partial_implementation": true,
+                "notes": "While the property is recognized, values applied to it don't have any effect."
+              },
```

Comment:
It seems like we decided in a similar case to mark support as `false`. Maybe @Elchi3 remembers.

Reply:
You're right, we did.  Lemme make the appropriate changes!